### PR TITLE
Add explorer url env

### PR DESCRIPTION
**Pull request by KedziaPawel**

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -19,4 +19,5 @@ export default {
   isBeta,
   applicationId: isBeta ? 'io.goldwallet.wallet.testnet' : 'io.goldwallet.wallet',
   applicationName: isBeta ? 'Testnet Gold Wallet' : 'GoldWallet',
+  explorerUrl: EXPLORER_URL || 'http://explorer.bitcoinvault.global',
```
Comment (marek-siemieniuk-morawski) [inline config/index.js:22]: shouldn't be `https`?

File: .env.example [hunk 2]
```diff
@@ -16,4 +16,7 @@ PORT=50002
 ELECTRUM_X_PROTOCOL_VERSION=2.0
 
 # true, false
-IS_BETA=false
\ No newline at end of file
+IS_BETA=false
+
+# http://explorer.bitcoinvault.global (mainnet) http://18.196.199.105 (testnet)
+EXPLORER_URL=http://explorer.bitcoinvault.global
```
Comment (marek-siemieniuk-morawski) [inline .env.example:22]: I'd add missing empty lines at end of files

**Review by marek-siemieniuk-morawski — APPROVED** [review]

File: .env.example (hunk 2 above)
Comment (marek-siemieniuk-morawski) [reply .env.example:22]: Also, `https` here imo

Comment (KedziaPawel) [reply .env.example]: done

**Review by KedziaPawel — COMMENTED** [review]
done

**Pull request merged**